### PR TITLE
Allow missingInterpolationHandler to be provided as t() option

### LINKED ISSUE #1117: Specify missingInterpolationHandler via t(_, options)
I've got a case where I'd like to detect missing interpolations for specific calls to `t()`. The newly added `missingInterpolationHandler` option would've have been perfect for this if it had been possible to override via the `t()` `options` argument, but unfortunately this is currently not the case.

From what I can see [this block](https://github.com/i18next/i18next/blob/0de7e5683565bcda8654aed5e9a59bf1ba916eaa/src/Interpolator.js#L97-L98) would need to be modified to first check if `missingInterpolationHandler` has been provided in the data object.

Thanks to all contributors for a great library by the way!

### PR DESCRIPTION
**Pull request by mogelbrod**
Fixes #1117.

### CODE REVIEW

File: src/Translator.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ class Translator extends EventEmitter {
       // interpolate
       let data = options.replace && typeof options.replace !== 'string' ? options.replace : options;
       if (this.options.interpolation.defaultVariables) data = { ...this.options.interpolation.defaultVariables, ...data };
-      res = this.interpolator.interpolate(res, data, options.lng || this.language);
+      res = this.interpolator.interpolate(res, data, options.lng || this.language, options);
```
Comment (mogelbrod) [inline src/Translator.js]: Not entirely satisfied with the additional argument here, but I also don't wish to risk breaking backwards compatibility.

**Comment by jamuhl** [comment]
already late here....will merge this tomorrow and publish an updated version to npm

**Comment by jamuhl** [comment]
Thank you for taking the time to provide this PR. Was merged and published to npm in i18next@11.10.0.

And don’t forget to star this repo. Make a tweet or have a look at our https://locize.com -> there are many ways to help this project :pray:

**Comment by mogelbrod** [comment]
Thanks for the quick responses, much appreciated! ⭐️

**Pull request merged**